employee/Employee/views.py

```diff
@@ -0,0 +1,50 @@
+from django.shortcuts import render
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+
+@method_decorator(csrf_exempt, name='dispatch')
```

Comment:
To free all HTTP methods like Get, post, put, delete  from CSRF protection. So I used @csrf_exempt decorator only for views so that application is secured too.

Reply:
@Raghugowd The answer what you've given is actually for why you've used the `csrf_exempt` decorator(anyways that would've been my next question :) ).
But my previous question still remains unanswered.